src/kernels/composable_kernel/include/tensor_operation/gridwise_gemm_xdlops.hpp

```diff
@@ -171,11 +169,7 @@ struct GridwiseGemmTransposedANormalBNormalCXdlops_v1
         __shared__ Float p_b_block_double[2 * b_block_space];
 
         // register allocation for output
-        AccFloat p_c_thread[c_k_thread_mtx_desc.GetElementSpace()];
-
-        // zero out threadwise output
-        threadwise_matrix_set_zero(c_k_thread_mtx_desc, p_c_thread);
-        blockwise_gemm.XdlopsMatrixCSetZero();
+        auto c_thread_vec = blockwise_gemm.GetOutputVec();
```

Comment:
Is c_thread_vec initialized to be 0 here? If so, it's better to indicate it's 0-initialization from the function name

Reply:
+1
`GetOutputVec` would 0 initialize `c_thread_vec`, so it's better to name it properly.

Reply:
Also, I suggest to use work ```Make``` instead of ```Get```, because it's creating a runtime variable

Reply:
The output vector is created at compiler time, right?

Reply:
Yes, on ISA, it's created on compile time. From C++ side, it would looks like it's creating an variable from nowhere, and give it 0-value. So I think ```Make``` or ```Create``` is better than ```Get```